getting_started/intel_debug/suggestions.rst

```diff
@@ -131,4 +131,12 @@ following command to extract DSP traces:
 
 .. code-block:: bash
 
-   sof-logger -t sof-tgl.ldc
\ No newline at end of file
+   sof-logger -t sof-tgl.ldc
```

Comment:
I think you need a `-l` option before the `.ldc` file?

Reply:
I think you are right, I will fix it while at it. I'm not sure why the change is registered, probably because the new lines.

Reply:
Good that it grabbed the line... this needs to be fixed (although unrelated to your patch).